### PR TITLE
Fix table horizontal scrolling

### PR DESCRIPTION
**Pull request by aroralakshya**
Changed files:
	themes/original/css/common.css.php
	themes/pmahomme/css/common.css.php
Added style definition for .responsivetable element to prevent horizontal overflow
Signed-Off-By: Lakshay arora (b16060@students.iitmandi.ac.in)

Before submitting pull request, please check that every commit:

- [x] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

File: themes/original/css/common.css.php [hunk 1]
```diff
@@ -3323,6 +3323,10 @@
     display: none;
 }
 
+.responsivetable {
+    overflow-x: auto !important;
```
Comment (mauriciofauth) [inline themes/original/css/common.css.php]: Using `!important` is generally not recommended. Could you solve it differently?

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

File: themes/original/css/common.css.php (hunk 1 above)
Comment (nulll-pointer) [reply themes/original/css/common.css.php]: It works without important, i just added it for safety reasons.

Comment (nulll-pointer) [reply themes/original/css/common.css.php]: i could remove it if you suggest.

**Comment by mauriciofauth** [comment]
Merged, thanks for your contribution!

**Pull request merged**